routes/web.php

```diff
@@ -14,15 +15,22 @@
 | contains the "web" middleware group. Now create something great!
 |
 */
+
+Route::controller(ShowController::class)->group(function() {
+    Route::get('/index', 'index')->name('index');
+    Route::get('/index/article/{id}', 'show')->name('article.show');
+});
+
+
 Auth::routes();
 Route::group(['middleware'=> 'auth'],function(){
     Route::controller(ArticleController::class)->group(function() {
```

Comment:
こんな感じで一旦やってみようか。くどいけどあえてArticleはArticleでまとめるスタイル

```suggestion
    Route::controller(AdminDashboardController::class)->group(function() {
            Route::get('/dashboard', 'dashboard')->name('admin.dashboard');
    });
   
    Route::controller(AdminArticleController::class)->group(function() {
            Route::get('/dashboard/article/create', 'create')->name('admin.article.create');
            Route::post('/dashboard/article/create', 'store')->name('admin.article.store');
            Route::patch('/dashboard/article/{id}/update', 'update')->name('admin.article.update');
            Route::get('/dashboard/article/{id}/edit', 'edit')->name('admin.article.edit');
            Route::get('/dashboard/article/{id}', 'show')->name('admin.article.show');
            Route::delete('/dashboard/article/{id}/destroy', 'destroy')->name('admin.article.destroy');
    });
```

Reply:
修正済。